src/Core/Transform/CMakeLists.txt

```diff
@@ -29,15 +29,19 @@ compare_to_baseline(EXAMPLE_NAME GlobalRegistrationTwoImagesAffine
     output
   )
 
-add_example(GlobalRegistrationTwoImagesBSpline)
-compare_to_baseline(EXAMPLE_NAME GlobalRegistrationTwoImagesBSpline
-  BASELINE_PREFIX GlobalRegistrationTwoImagesBSpline
-  )
-
 add_example(MutualInformationAffine)
 compare_to_baseline(EXAMPLE_NAME MutualInformationAffine
   BASELINE_PREFIX
     fixed
     moving
     output
   )
+
+if ()
```

Comment:
@mseng10  What is this if() for?

Please fix this up based on the PR that I have in place here.